src/browser/api/application.js

```diff
@@ -1008,8 +1008,9 @@ function createAppObj(uuid, opts, configUrl = '') {
                 } else {
                     log.writeToLog(1, `receiving net error ${errorCode} for ${opts.uuid}`, true);
                     if (!coreState.argo['noerrdialog'] && configUrl) {
+                        const errorMsgForDialog = errorDescription || `error code ${ errorCode }`;
```

Comment:
`errorDescription` is passed into the `did-fail-load` listener: https://github.com/HadoukenIO/core/pull/66/files#diff-a08b93ff0f46d39099ebdc2bb86958a1L1003